multirun/def.bzl

```diff
@@ -103,8 +103,8 @@ def _command_impl(ctx):
         for k, v in ctx.attr.raw_environment.items()
     ]
     str_args = [
-        "%s=%s" % (k, shell.quote(v))
-        for k, v in ctx.attr.arguments.items()
+        "%s" % shell.quote(v)
```

Comment:
This does not work - escaping the "key" makes it a positional parameter and obviously that breaks things.

Reply:
Hmm, we are using this already in our code-base as follows:

```
command(
    name = "gazelle-ci",
    arguments = [
        "-bazel_run",
        "-mode",
        "diff",
    ],
    command = ":gazelle",
)
```

and that works. Or are you referring to a different case?

Reply:
I think the following will not work, but I think it would if passed via command line directly:
```bzl
command(
    name = "gazelle-ci",
    arguments = [
        "-bazel_run",
        "-mode=diff",
    ],
    command = ":gazelle",
)
```

It didn't work when it was `shell.quote("%s=%s" % (k, v))`.

Reply:
At least in this case it is working. When having `set -x` I can see `exec env ./gazelle -bazel_run -mode=diff`

But even if for some reason there is a case where that usage would break if we make this a separate argument to the existing dict usage I think it is acceptable. What do you think=

Reply:
Hm, if it is working then it is probably ok? I don't use this rule so cannot test it.